### PR TITLE
Feature: add landing page

### PR DESCRIPTION
**Pull request by oyershov**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -45,6 +45,8 @@ class AppLayout extends React.Component<Props, {}, {}> {
             locale,
         } = this.props;
         const { lang, messages } = locale;
+        const shouldRenderFooter = window.location.pathname !== '/';
```
Comment (mod) [inline src/App.tsx]: I want the footer everywhere

File: src/assets/images/landing/FifthBG.svg [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<svg width="1920" height="500" viewBox="0 0 1920 500" fill="none" xmlns="http://www.w3.org/2000/svg">
+<mask id="mask0" mask-type="alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="1920" height="500">
+<rect width="1920" height="500" fill="#171C29"/>
+</mask>
+<g mask="url(#mask0)">
+<rect width="1920" height="500" fill="#171C29"/>
+<g opacity="0.15">
```
Comment (mod) [inline src/assets/images/landing/FifthBG.svg]: when naming picture it's better to describe what is it
illustration_monkey for example

File: src/assets/images/landing/features/Icon1.svg [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+<path d="M37.9202 14.7364L28.5048 5.32105C27.9794 4.79585 27.1278 4.79601 26.6026 5.32136C26.3505 5.57356 26.2089 5.91544 26.2088 6.27207V10.3072H16.7934C16.0505 10.3072 15.4483 10.9094 15.4483 11.6523C15.4483 12.3952 16.0505 12.9974 16.7934 12.9974H27.5538C28.2967 12.9974 28.8989 12.3952 28.8989 11.6523V9.51895L35.0674 15.6874L28.8989 21.8558V19.7225C28.8989 18.9796 28.2967 18.3774 27.5538 18.3774H12.7582V14.3423C12.7581 13.5994 12.1557 12.9974 11.4129 12.9975C11.0564 12.9976 10.7144 13.1392 10.4622 13.3914L1.04692 22.8067C0.521796 23.332 0.521796 24.1834 1.04692 24.7086L10.4623 34.124C10.7145 34.3763 11.0565 34.5181 11.4132 34.5181C11.59 34.5187 11.7652 34.4839 11.9284 34.4159C12.4309 34.2075 12.7584 33.7171 12.7582 33.1731V29.1379H22.1736C22.9165 29.1379 23.5187 28.5357 23.5187 27.7928C23.5187 27.05 22.9165 26.4478 22.1736 26.4478H11.4132C10.6704 26.4478 10.0682 27.05 10.0682 27.7928V29.9261L3.89974 23.7577L10.0682 17.5893V19.7225C10.0682 20.4654 10.6704 21.0676 11.4132 21.0676H26.2088V25.1028C26.2089 25.8456 26.8113 26.4477 27.5541 26.4475C27.9106 26.4474 28.2526 26.3058 28.5048 26.0537L37.9202 16.6383C38.4452 16.113 38.4452 15.2616 37.9202 14.7364Z" fill="#3C78E0"/>
+</g>
```
Comment (mod) [inline src/assets/images/landing/features/Icon1.svg]: same we need to know what is it
also you don't have an icon pack / font ?

File: src/containers/MarketsTable/MarketsTable.pcss [hunk 4]
```diff
@@ -0,0 +1,165 @@
+.pg-markets-table {
```
Comment (mod) [inline src/containers/MarketsTable/MarketsTable.pcss]: CSS should not go in containers, make a component
Call it a tickerTable

File: src/assets/images/landing/features/Icon1.svg (hunk 3 above)
Comment (oyershov) [reply src/assets/images/landing/features/Icon1.svg]: Renamed, we should speak to designer about pack / font

**Pull request merged**